st2mistral/functions/stackstorm.py

```diff
@@ -18,17 +18,25 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
 from st2mistral.utils import http
 
+from st2mistral import config
+config.register_opts()
```

Comment:
Okay, fixed in https://github.com/StackStorm/st2mistral/pull/30/commits/c12ff85045d7bd692ed662f9ad5fba73895e9fef